torch/csrc/Module.cpp

```diff
@@ -908,6 +908,32 @@ PyObject* initModule() {
   py_module.def("_demangle", &c10::demangle);
   py_module.def("_log_api_usage_once", &LogAPIUsageOnceFromPython);
 
+  py::enum_<c10::LogLevel>(py_module, "LogLevel")
+        .value("FATAL",   c10::LogLevel::Fatal)
+        .value("ERROR",   c10::LogLevel::Error)
+        .value("WARNING", c10::LogLevel::Warning)
+        .value("INFO",    c10::LogLevel::Info);
+
+  py_module.def(
+    "set_c10_log_level",
+    c10::setLogLevel,
+    R"(
+set_c10_log_level(level: LogLevel) -> None
+
+Sets the threshold for the c10 logger to `level`.
+)");
+
+  py_module.def(
+    "get_c10_log_level",
+    c10::getLogLevel,
+    R"(
+get_c10_log_level() -> LogLevel
+
+Gets the threshold for the c10 logger.
```

Comment:
-EPARSE (not sure what is "threshold for the logger", perhaps we should make it a bit more verbose)
```suggestion
Gets the current log level  for the low level Pytorch logging facility.
```